js_modules/dagster-ui/packages/ui-core/src/assets/AssetsOverview.tsx

```diff
@@ -0,0 +1,337 @@
+type CountPerCodeLocation = {
+  repoAddress: RepoAddress;
+  assetCount: number;
+};
+
+function buildAssetCountBySection(assets: AssetTableFragment[]): AssetCountsResult {
+  const assetCountByOwner: Record<string, number> = {};
+  const assetCountByComputeKind: Record<string, number> = {};
+  const assetCountByGroup: Record<string, number> = {};
+  const assetCountByCodeLocation: Record<string, number> = {};
+
+  assets
+    .filter((asset) => asset.definition)
+    .forEach((asset) => {
+      const assetDefinition = asset.definition!;
+      assetDefinition.owners.forEach((owner) => {
+        const ownerKey = owner.__typename === 'UserAssetOwner' ? owner.email : owner.team;
+        assetCountByOwner[ownerKey] = (assetCountByOwner[ownerKey] || 0) + 1;
+      });
+
+      const computeKind = assetDefinition.computeKind;
+      if (computeKind) {
+        assetCountByComputeKind[computeKind] = (assetCountByComputeKind[computeKind] || 0) + 1;
+      }
+
+      const groupName = assetDefinition.groupName;
+      const locationName = assetDefinition.repository.location.name;
+      const repositoryName = assetDefinition.repository.name;
+
+      if (groupName) {
+        const metadata: GroupMetadata = {
+          groupName,
+          repositoryLocationName: locationName,
+          repositoryName,
+        };
+        const groupIdentifier = JSON.stringify(metadata);
+        assetCountByGroup[groupIdentifier] = (assetCountByGroup[groupIdentifier] || 0) + 1;
+      }
+
+      const stringifiedCodeLocation = buildRepoPathForHuman(repositoryName, locationName);
+      assetCountByCodeLocation[stringifiedCodeLocation] =
+        (assetCountByCodeLocation[stringifiedCodeLocation] || 0) + 1;
+    });
+
+  const countPerAssetGroup = Object.entries(assetCountByGroup).map(([groupIdentifier, count]) => ({
+    assetCount: count,
+    groupMetadata: JSON.parse(groupIdentifier),
+  }));
+
+  return {
+    countsByOwner: assetCountByOwner,
+    countsByComputeKind: assetCountByComputeKind,
+    countPerAssetGroup,
+    countPerCodeLocation: Object.entries(assetCountByCodeLocation).map(([key, count]) => {
+      return {repoAddress: repoAddressFromPath(key)!, assetCount: count};
+    }),
+  };
+}
+
+interface AssetOverviewCategoryProps {
+  children: React.ReactNode;
+  assetsCount: number;
+}
+
+function getGreeting(timezone: string) {
+  const hour = Number(
+    new Date().toLocaleTimeString('en-US', {
+      hour: '2-digit',
+      hourCycle: 'h23',
+      timeZone: timezone === 'Automatic' ? browserTimezone() : timezone,
+    }),
+  );
+  if (hour < 12) {
+    return 'Good morning';
+  } else if (hour < 18) {
+    return 'Good afternoon';
+  } else {
+    return 'Good evening';
+  }
+}
+
+const CountForAssetType = ({children, assetsCount}: AssetOverviewCategoryProps) => {
+  return (
+    <Box
+      flex={{direction: 'row', justifyContent: 'space-between'}}
+      style={{width: 'calc(33% - 16px)'}}
+    >
+      <Box>{children}</Box>
+      <Box>
```

Comment:
do you need the extra box here? Doesn't seem like it's doing anything

Reply:
good call, we do not need this. Removed it